tests/test_ignore_paths.py

```diff
@@ -0,0 +1,57 @@
+"""Tests for --ignore-paths support.
```

Comment:
Ad these tests to existing tests. `build_manifest` is tested by the `test_git_repo.py` tests.